html_to_playlist.py

```diff
@@ -1,63 +1,82 @@
 import argparse
-
 import pandas as pd
 
+# local
 import utils
+from spotify_abs_cls import SpotifuHandler
 
+# constants
 IS_PUBLIC_PLAYLIST = True
 
-
 def parse_args():
     parser = argparse.ArgumentParser(fromfile_prefix_chars='@')
-    parser.add_argument('--html-file', '-f', type=str, help='HTML file path',
-                        default='bidud_party.html', )
-    parser.add_argument('--spotify-yaml', '-s', help='Spotify credentials file',
+    parser.add_argument('--spotify-yaml-path', '-s', help='Spotify credentials file',
                         default='./spotify.yaml', type=str)
+    parser.add_argument('--html-file-path', '-f', type=str, help='HTML file path',
+                        default='bidud_party.html', )
     parser.add_argument('--playlist-name', '-n', type=str, default='Bidud Party Tests', help='Playlist name')
     parser.add_argument('--playlist-description', '-d', type=str, default='Yayz', help='Playlist description')
     return parser.parse_args()
 
 
-def parse_html(html_file, spotify_yaml, playlist_name, playlist_description):
-    with open(html_file, 'rb') as fid:
-        html_text = fid.read()
-    tables = pd.read_html(html_text)
-    playlist_table = tables[0]
-    playlist_table = playlist_table.rename(columns=playlist_table.iloc[0])
-    playlist_table = playlist_table.drop(playlist_table.index[0])
-    # with open(html_file[:-5] + '_playlist.txt', 'wb') as fid:
-    #     for index, row in playlist_table.iterrows():
-    #         fid.write("{} - {}\n".format(row['Artist'], row['Title']).encode("UTF-8"))
-    # print(row['Artist'], row['Title'])
+class HtmlToPlaylist(SpotifuHandler):
+    """
+    Create playlist from streaming history, with defined time ranges.
```

Comment:
This comment is not relevant to this class

Reply:
True, fixing.